examples/objective-c-ios/objective-c-ios/AppDelegate.m

```diff
@@ -59,6 +65,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 //    config.enabledErrorTypes.unhandledExceptions = YES;
 //    config.enabledErrorTypes.machExceptions = YES;
     
+    /**
+     * To enable network breadcrumbs, add the BugsnagRequestMonitor plugin to your config.
+     */
+//    [config addPlugin:[BugsnagRequestMonitor new]];
```

Comment:
In general I recommend avoiding `+new`, and using `+alloc` and `-init`; that's what Apple do in their example / sample code.